archesproject/arches-dev-dependencies
# removes post-css-preset-env

This is cruft leftover from the initial webpack implementation

To test:

1. create a project
2. remove node_modules folder and yarn.lock
3. update package.json to point to this branch for devDependencies
4. if it exists in the project, delete  `postcss.config.js`
5. run `yarn install`
6. build the frontend asset bundle
7. navigate the application
8. notice no change in CSS

this is blocking https://github.com/archesproject/arches/pull/10953
